app/Services/EventsAppService.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace App\Services;
+
+use GuzzleHttp\Client;
+use GuzzleHttp\Exception\RequestException;
+use GuzzleHttp\Psr7\MultipartStream;
+
+class EventsAppService {
```

Comment:
Why do you have separated class if you already have service class for API? This should be merged into one class and methods here should be properly fixed